x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -24061,7 +24061,7 @@
     "xpack.security.management.users.usersTitle": "Utilisateurs",
     "xpack.security.management.usersTitle": "Utilisateurs",
     "xpack.security.navControlComponent.accountMenuAriaLabel": "Menu Compte",
-    "xpack.security.navControlComponent.editProfileLinkText": "{profileOverridden, select, true{Préférences} other{Profil}}",
+    "xpack.security.navControlComponent.editProfileLinkText": "Profil",
```

Comment:
__nit:__ if you happened to update translation files manually then it'd make sense to revert these changes and run `node scripts/i18n_check.js --fix` to remove outdated translations instead. Updated translations will be eventually provided by our translators team.

Reply:
Which looks like it just removed the line entirely, is that correct?

Reply:
Yep, that's correct!